versioned_docs/version-2.x/advanced/payments/stripe.mdx

```diff
@@ -33,8 +33,19 @@ With API keys found
 # API Keys from https://dashboard.stripe.com/account/apikeys
 FRONT_COMMERCE_WEB_STRIPE_PUBLISHABLE_KEY=pk_xxxxxx
 FRONT_COMMERCE_STRIPE_SECRET_KEY=sk_xxxxxx
+
+# Only for Magento2 / Magento2 B2B integrations
+FRONT_COMMERCE_STRIPE_ENDPOINT_SECRET=whsec_xxxxxx
 ```
 
+:::info
+
+In order to retrieve the enpoint secret, you will need to create a webhook in
```

Comment:
Typo:
```suggestion
In order to retrieve the endpoint secret, you will need to create a webhook in
```

---

**issue:** the webhook URL isn't documented. Neither here or in the section below

**suggestion:** provide an URL example of what FC expects to have been configured as webhook URL

Reply:
Good catch, I added an example in the Magento2 steps section.